### PR TITLE
Chart changes with additional configurations

### PR DESCRIPTION
**Pull request by nshankar13**
**Description**:
Parameterize additional chart values

**Affected area**:

- New Functionality      [ ]
- Documentation          [ ]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Performance            [ ]
- Other                  [X ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

### CODE REVIEW

File: charts/osm/values.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ OpenServiceMesh:
     protocol: http
     token:
     role: openservicemesh
+  connectVault: true
```
Comment (shashankram) [inline charts/osm/values.yaml:22]: A comment on what this does will be helpful.

**Review by shashankram — APPROVED** [review]

File: charts/osm/values.yaml (hunk 1 above)
Comment (nojnhuh) [reply charts/osm/values.yaml:22]: Even better than a comment, an entry in values.schema.json with a description.

File: charts/osm/templates/fluentbit-configmap.yaml [hunk 2]
```diff
@@ -21,8 +21,13 @@ data:
       match      *
       regex       message /"level":"error"/
     [OUTPUT]
-      Name    stdout
+      Name    {{ .Values.OpenServiceMesh.fluentbitOutput }}
```
Comment (ritazh) [inline charts/osm/templates/fluentbit-configmap.yaml:24]: Maybe this whole section under `[OUTPUT]` should be parameterized as each output plugin will have its own configurations. 
WDYT @SanyaKochhar

Comment (ritazh) [reply charts/osm/templates/fluentbit-configmap.yaml:24]: On second thought, it's good to have the ability to set these fields individually.

**Review by ksubrmnn — APPROVED** [review]

**Review by snehachhabria — APPROVED** [review]

**Pull request merged**